setup.py

```diff
@@ -27,7 +27,7 @@ def package_files(directory):
     "torchvision>=0.9.0",
     "torch>=1.8.1",
     "argparse>=1.4.0",
-    "yahp>=0.0.10",
+    "yahp>=0.0.11",
```

Comment:
Don't worry about changing Yahp versions, I'll handle that separately just so I can ensure to track it properly

Reply:
Otherwise looks good to me, but ditto on hanlins requests

Reply:
```suggestion
    "yahp>=0.0.10",
```

Reply:
I'll keep it here just so we don't merge this in until new yahp is released (this diff should disappear once you add that commit in).

Reply:
Yeah, fyi YAHP is already on `0.0.11`, so that isn't the right number to ensure it updates with YAHP either.  I think I'd rather just track all version numbers independently for now to keep it lower overhead for me.

Reply:
Marked with a new label, but as this is a backwards compatible change you should be fine to merge into dev